.github/workflows/ci.yml

```diff
@@ -37,12 +37,12 @@ jobs:
       -
         name: Set tag for release
         if: ${{ matrix.version == 'release' }}
-        run: printf 'TAG=' && curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' >> $GITHUB_ENV
+        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/TAG=\1/' >> $GITHUB_ENV
 
       -
         name: Build and push
         uses: docker/build-push-action@v2
         with:
           push: true
-          build-args: VERSION=${{ matrix.version }}
+          build-args: VERSION=${{ env.TAG }}
```

Comment:
This won't work because the TAG for master is latest. I guess we'll need two additional steps to set build arg vars